client/my-sites/controller.js

```diff
@@ -158,19 +160,19 @@ function renderSelectedSiteIsDomainOnly( reactContext, selectedSite ) {
 
 function isPathAllowedForDomainOnlySite( path, slug, primaryDomain ) {
 	const allPaths = [
-		domainManagementAddGSuiteUsers,
```

Comment:
It needs to be `emailManagementAddGSuiteUsers`, otherwise future work on independent G Suite subs will not be helped at all by these changes. It will be more clear once I correctly remove all other references to the `domains/manage/email` path